interfaces/builtin/network_setup_observe.go

```diff
@@ -32,6 +32,20 @@ const networkSetupObserveBaseDeclarationSlots = `
 const networkSetupObserveConnectedPlugAppArmor = `
 # Description: Can read netplan configuration files
 
+# Allow use of the netplan binary from the base snap. With this interface, this 
+# is expected to be able to only get information about the current network 
+# configuration and not generate or apply it like is allowed with 
+# network-setup-control.
```

Comment:
Which part doesn't make sense to you? All of the accesses added to `-observe` here are for reading or executing the netplan binaries from the base snap. This enables clients like:

```bash
out=$(netplan info)
# parse ip from netplan info output
```

instead of having to bundle all of netplan within their own snap which is awkward and annoying

Reply:
I somehow misread the comment apparently